plugins/seagate/seagate-nvme.c

```diff
@@ -126,6 +132,20 @@ static char *log_pages_supp_print(__u32 pageID)
 	}
 }
 
+static int stx_is_jag_pan(char *devMN)
+{
+    int match_found = 1; /* found = 0, not_found = 1 */
+
+    for (int i = 0; i < STX_NUM_LEGACY_DRV; i++) 
+    {
```

Comment:
Oh man I just realized the whole plugin code is a using all sorts of coding style. I would really love if someone cleans it up. While checkpatch.pl is not perfect it helps out getting the most annoying offenders fixed.

For example:

```shell
$ ~/work/linux/scripts/checkpatch.pl -f plugins/seagate/seagate-nvme.c
[...]
ERROR: space required before the open parenthesis '('
#56: FILE: plugins/seagate/seagate-nvme.c:56:
+       switch(pageID) {
[...]
```

And I am starting to think we might do this with the approach from https://github.com/linux-nvme/nvme-cli/pull/1278 but just for the plugins.